rfcs/build-system/01_convergence_devstack.md

```diff
@@ -0,0 +1,191 @@
+# WIP | FUI Convergence - devstack/DX
+
+> **tags:** pitch, convergence
+
+> For instructions on shaping a project see here: [Shaping a Project](/kX02SXVbS6KzMOQd56i6Cg)
+>
+> Ported from [hackmd](https://hackmd.io/MiS9ISfjRXqmd6XmxpyMaA)
+
+### Problem
+
+Based on [high level repo issues notes](https://hackmd.io/0TIUjU8_T1Ga2Wjez4IiSQ), we need to address most crucial ones sooner than later to enable good enough DX for both inside/outside contributors.
+
+> Good enough DX:
+>
+> - rapid feedback loop
+> - one common way how to do/run things
+> - docs (reflected within tooling -> `yarn test --help` should just work)
+
+### Appetite
+
+As an initial step lets focus on resolve following (smallest possible area):
+
+1. need to build before run/test/demo
+1. unified TSConfigs
+1. unified way how to transpile javascript
+1. solution for convergence/whole repo
+
+### Solution
+
+> show don't tell / TL;DR
+> ![](https://i.imgur.com/bDwWu9v.png)
+
+#### 1. need to build before run/test/lint/demo
```

Comment:
The UMD bundles are primarily needed for codepens.

Agree that codesandbox makes the less of a necessity - but I believe before we consider dropping UMD output, the website "export to codepen" examples should all be converted to codesandbox, otherwise stuff breaks.

Reply:
@dzearing  other question - what browsers do we need to support for those examples? ( for eg, if we support evergreen only, we can just use ESM in demos as well )

Reply:
Depends on whether we intend to support IE 11 for new packages in general. If yes, unfortunately I think we'd also need to support IE 11 for examples.

Reply:
> Depends on whether we intend to support IE 11 

Who can answer this ?